blueprints/tangle-avs-blueprint/src/main.rs

```diff
@@ -0,0 +1,130 @@
+pub mod runner;
```

Comment:
Tangle AVS blueprint is exclusively for Eigenlayer execution. It should not have a SubstrateEventListener IMO. It doesn't totally make sense to have a tangle operator run a tangle validator.. they'd just already be running a validator we wouldn't need a blueprint for it on Tangle, right?

Reply:
I agree, I'm not 100% sure what exactly we want the Tangle AVS to do. I am removing it from this PR for now, since it is not going to be built out here anyway.

Reply:
For starters @Tjemmmic, we just want the AVS to run the validator client and register the keys involved that the validator will use. This should be sufficient to get on the testnet.

The secondary things that we will want to find process for are:
 - Registering for validator using tangle extrinsics (how will the AVS get a balance? can we do a cross-chain TX using hyperlane and gas subsidy?)
 - Implementing cross-chain messaging/eventually slashing logic (receive message from hyperlane bridge to the AVS contract).

These secondary things should come _after_ we get the first part done.